app/views/users/_notification_preferences.html.erb

```diff
@@ -10,32 +10,27 @@
     <p class="form-control-static"><strong>All Users</strong></p>
     <div class="checkbox">
       <label for="prefs[users][new_comment]">
-        <%= hidden_field_tag 'prefs[users][new_comment]', false %>
```

Comment:
This does indeed remove the duplicate ids from the HTML but the page now no longer functions properly. Your changes do not save if these fields are removed. The controller would need to be updated as well